manage-users/README.md

```diff
@@ -1 +1,124 @@
-# HomePage Component
\ No newline at end of file
+# Deployment and installation
+With this configuration, you can use the ent cli (https://dev.entando.org/next/docs/reference/entando-cli.html) to perform the full deployment sequence:
+
+### Setup the project directory
+1. Prepare the bundle directory: `cp -r bundle_src bundle`
+2. Initialize the project: `ent prj init`
+3. Initialize publication: `ent prj pbs-init` (requires the git bundle repo url)
+
+### Publish the bundle.
+1. Build: `ent prj fe-build -a` (build the frontend, including changes from bundle_src)
+2. Publish: `ent prj fe-push`
+3. Deploy (after connecting to k8s): `ent prj deploy`
+4. Install the bundle via 1) App Builder, 2) `ent prj install`, or 3) `ent prj install --conflict-strategy=OVERRIDE` on subsequent installs.
+5. Iterate steps 1-4 to publish new versions.
+
+# Development Tips
+The Manage Users MFE requires the authenticated user to have access to manage users within Keycloak. If you're running the MFE locally with the default keycloak realm loaded, you'll need to grant this permission to your user before you can see the list of users. Go to Users → <user-name> → Role Mappings → realm-management and grant the `view-users` (or a higher level role) to the user.
+
+# Manage Users
+
+This application was generated using JHipster 6.3.0, you can find documentation and help at [https://www.jhipster.tech/documentation-archive/v6.3.0](https://www.jhipster.tech/documentation-archive/v6.3.0).
```

Comment:
is it intended to keep the JHipster readme here since we did not keep it for the previous MS?